### PR TITLE
Warn about the current state of development in the pull request template

### PR DESCRIPTION
**Pull request by rniii**
Also a few readability fixes to the grammar

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
 <!--
 Thanks a lot for making a contribution to SFML! 🙂
 
-Before you create the pull request, we ask you to check the follow boxes. (For small changes not everything needs to ticked, but the more the better!)
+Please make sure you are targetting the correct branch. No more features are planned for the 2.x branches! (See State of Development in the readme)
```
Comment (eXpl0it3r) [inline .github/pull_request_template.md:4]: Maybe `readme` could be a link to the readme

Comment (rniii) [reply .github/pull_request_template.md]: I'm avoiding making the line too long, so I changed the text and linked to the section

**Comment by eXpl0it3r** [comment]
SFMLBuildMaster: Build this please.

**Review by eXpl0it3r — APPROVED** [review]
Looks good to me 🙂

**Comment by eXpl0it3r** [comment]
Thank you for adding this!

**Pull request merged**